src/GrommetSwagger.js

```diff
@@ -37,19 +37,35 @@ export default class GrommetSwagger extends Component {
     }
   }
 
+  getOrigin = (htmlHost, configHost, configUrl) => {
+    const parser = document.createElement('a');
+    parser.href = configUrl;
+    const origin = htmlHost || configHost || configUrl.origin;
+
+    if (origin.indexOf('http') === -1) {
```

Comment:
This is vulnerable to a host name like `resthttp.my.com`. Perhaps we should do something like `origin.slice(0, 4) !== 'http'`?

Reply:
Awesome, didn't realize slice was also a String method.